include/openssl/ssl.h

```diff
@@ -1493,8 +1493,9 @@ __owur int SSL_CTX_dane_mtype_set(SSL_CTX *ctx, const EVP_MD *md,
 __owur int SSL_dane_enable(SSL *s, const char *basedomain);
 __owur int SSL_dane_tlsa_add(SSL *s, uint8_t usage, uint8_t selector,
                              uint8_t mtype, unsigned char *data, size_t dlen);
-__owur int SSL_get0_dane_authority(SSL *s, X509 **mcert, EVP_PKEY **mspki);
-__owur int SSL_get0_dane_tlsa(SSL *s, uint8_t *usage, uint8_t *selector,
+__owur int SSL_get0_dane_authority(const SSL *s, const X509 **mcert,
```

Comment:
Curious, why are these `int`? If the structures are optional, isn't returning `NULL` sufficient?